data/Iyanu1396.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "IyanuOluwa",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend Developer | Javascript ",
+  "avatar": "https://avatars.githubusercontent.com/u/108562866?s=400&u=63b79f7eb10da91446c3af03ee453029795889f1&v=4",
```

It's best to use this link as it's shorter and updates automatically.
```suggestion
  "avatar": "https://github.com/Iyanu1396.png",
```